.changeset/wise-falcons-allow.md

```diff
@@ -0,0 +1,5 @@
+---
+"@folks-finance/xchain-sdk": patch
+---
+
+Added random adapter selection in reverse msg
```

Comment:
Should delete this changeset

Reply:
9e329370dae038987f9bf9ddb2e8a52229ed2aeb